src/main/java/de/dhbw/karlsruhe/students/mailflow/core/domain/email/Email.java

```diff
@@ -0,0 +1,69 @@
+package de.dhbw.karlsruhe.students.mailflow.core.domain.email;
+
+import java.util.List;
+import java.util.Set;
+
+import de.dhbw.karlsruhe.students.mailflow.core.domain.common.models.AggregateRoot;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.value_objects.EmailMetadata;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.value_objects.Attachment;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.value_objects.EmailId;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.value_objects.Header;
+
+/**
+ * Representation of an e-mail as AggregateRoot
+ */
+public final class Email extends AggregateRoot<EmailId> {
+    private EmailMetadata emailMetadata;
+    private boolean isRead;
+    private String content;
+    private List<Header> headers;
+    private Set<Attachment> attachments;
+
+    private Email(EmailId id, String content, List<Header> headers, EmailMetadata emailMetadata,
```

Comment:
maybe add another constructor without attachment. not every e-mail has an attachment.
this.attachments = Collections.emptySet();
Maybe this is a option
[Link](https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#EMPTY_SET)